package.json

```diff
@@ -1,21 +1,28 @@
 {
   "name": "relay",
-  "version": "0.0.1",
+  "type": "module",
+  "engines": {
+    "node": ">=14.16"
+  },
+  "exports": "./dist/index.js",
   "scripts": {
-    "start": "ts-node src/index.ts"
+    "start_linux": "rm -rf ./dist && tsc && node ./dist/main.js",
+    "start_win": "del /s /q dist && tsc && node ./dist/main.js"
```

Comment:
why are you deleing stuff before running? tsc should be able to overwrite existing files

Reply:
TSC does not delete files if you rename, add folders, etc. caused me a runtime bug. This just clears dist before running.